#! /bin/sh /usr/share/dpatch/dpatch-run
## config-authproxy-socket.dpatch by  <David Bremner <bremner@unb.ca>>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: add support for an "authproxy_socket" configuration option

@DPATCH@
diff -urNad pushmi-1.0.0~/lib/Pushmi/Config.pm pushmi-1.0.0/lib/Pushmi/Config.pm
--- pushmi-1.0.0~/lib/Pushmi/Config.pm	2007-08-30 10:12:16.000000000 +0200
+++ pushmi-1.0.0/lib/Pushmi/Config.pm	2007-10-13 18:54:45.000000000 +0200
@@ -31,7 +31,7 @@
 sub memcached {
     return Cache::Memcached->new(
         {   'servers' =>
-                [ "127.0.0.1:" . (Pushmi::Config->config->{authproxy_port} || 7123) ],
+                [ Pushmi::Config->config->{authproxy_socket} || "127.0.0.1:" . (Pushmi::Config->config->{authproxy_port} || 7123) ],
             'debug' => 0
         }
     );
